web/src/i18n/locales/en/activities/costAllocate.yaml

```diff
@@ -50,6 +50,9 @@ ffp:
       Medicaid share.
   federalStateSplitInstruction:
     heading: Federal-State Split
+    detail: Select the match rate for Federal Financial Participation applicable
```

Comment:
It looks like we typically use the `>-` followed by a newline in our i18n yaml files for blocks of text, which trims extra spaces (if I'm interpreting the documentation correctly).

https://yaml.org/spec/1.2/spec.html#id2779048

Reply:
good catch